### ISSUE TITLE
Automation/Ease

### ISSUE THREAD

**Issue by algeboy** [equalspeak/EqualSpeak#1]
To be useful it will be necessary for TeX users to be able to load everything as a package and activate it simply.  It may also be that there should be two modes, one that builds in regular format and one that builds in screen-reader format.  

e.g.

Two files is not perhaps ideal so maybe a longer term option is to figure out how to embed both in one.  E.g. make it the alt text of HTML or at pdftooltip for PDF so that a screen reader can find the converted text but the original equations still display as expected.

```
\usepackage[html]{equalspeak}  % activates alt text for mathjax or something
\usepackage[tooltip]{equalspeak}  % screen reader format put into pdftooltip
\usepackage[screenreader]{equalspeak}  % activates screen reader format
```

**Comment by algeboy** [comment]
connecting with `accsupp` might be one option, though I have not managed to get that to work on anything
